src/main/java/uk/gov/companieshouse/chd/order/api/logging/LoggingUtils.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.chd.order.api.logging;
+
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+public class LoggingUtils {
+    
+    public static final String APPLICATION_NAMESPACE="chd-order-api";
+    
```

Comment:
This constant could become `private` given the `getLogger()` method? Not terribly important.

Reply:
true